R/fetch_inprogress.R

```diff
@@ -0,0 +1,169 @@
+#' Download in progress responses from a survey and import it into R
+#'
+#' Download in progress responses from a Qualtrics survey you own via API and import the survey directly into R.
+#'
+#' @inheritParams fetch_survey
+#'
+#' @seealso See <https://api.qualtrics.com/> for documentation on
+#' the Qualtrics API.
+#' @template retry-advice
+#' @importFrom lifecycle deprecated
+#' @export
+#' @examples
+#' \dontrun{
+#' # Register your Qualtrics credentials if you haven't already
+#' qualtrics_api_credentials(
+#'   api_key = "<YOUR-API-KEY>",
+#'   base_url = "<YOUR-BASE-URL>"
+#' )
+#'
+#' # Retrieve a list of surveys
+#' surveys <- all_surveys()
+#'
+#' # Retrieve a single survey
+#' mysurvey <- fetch_inprogress(surveyID = surveys$id[6])
+#'
+#' mysurvey <- fetch_inprogress(
+#'   surveyID = surveys$id[6],
+#'   save_dir = tempdir(),
+#'   start_date = "2018-01-01",
+#'   end_date = "2018-01-31",
+#'   limit = 100,
+#'   label = TRUE,
+#'   unanswer_recode = 999,
+#'   verbose = TRUE,
+#'   # Manually override EndDate to be a character vector
+#'   col_types = readr::cols(EndDate = readr::col_character())
+#' )
+#'
+#' }
+#'
+fetch_inprogress <- function(surveyID,
```

Comment:
```suggestion
fetch_in_progress <- function(surveyID,
```
Let's change the function name so it doesn't squash two words together.